client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/BackFromAbroad.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { AddCircle } from '@material-ui/icons';
+import { Controller, useFormContext } from 'react-hook-form';
+import { Collapse, Divider , IconButton , Typography } from '@material-ui/core';
+
+import Toggle from 'commons/Toggle/Toggle';
+import useFormStyles from 'styles/formStyles';
+import FormTitle from 'commons/FormTitle/FormTitle';
+import FieldName from 'commons/FieldName/FieldName';
+import FormRowWithInput from 'commons/FormRowWithInput/FormRowWithInput';
+import { Exposure , fieldsNames } from 'commons/Contexts/ExposuresAndFlights';
+
+import FlightsForm from './FlightsForm/FlightsForm';
+import useStyles from '../ExposuresAndFlightsStyles';
+
+const addFlightButton: string = 'הוסף טיסה לחול';
+
+interface Props {
+    wereFlights: boolean;
+    onExposuresStatusChange: (fieldName: any, value: any) => void; 
+    exposures: Exposure[];
+    handleChangeExposureDataAndFlightsField: (index: number, fieldName: string, value: any) => void; 
+    onExposureAdded: (wasConfirmedExposure: boolean, wasAbroad: boolean) => void;
+  	disableFlightAddition: boolean;
+}
+
+export const BackFromAbroad = (props: Props) => {
+	const { control , watch } = useFormContext();
+    const { fieldContainer } = useFormStyles();
+    const {
+        wereFlights,
+        onExposuresStatusChange, 
+        exposures,
+        handleChangeExposureDataAndFlightsField, 
+        onExposureAdded,
+		disableFlightAddition,
+    } = props;
+    const classes = useStyles();
+
+	  const watchWereFlights = watch(fieldsNames.wereFlights , wereFlights);
+
+    return (
+        <div className={classes.subForm}>
+            <FormTitle title='חזרה מחו״ל' />
+
+            <FormRowWithInput testId='wasAbroad' fieldName='האם חזר מחו״ל?'>
+				<Controller 
```

Comment:
fix indentation

Reply:
urghhhhhh all of the project is formatted in tabs and some pages are formatted in spaces (2 spaces) , I (hopefully) formatted the page to tabs now ([here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1017/commits/38cd482373be4af936cc820be053057d81097e38))